src/app/features/home/home.component.ts

```diff
@@ -1,15 +1,28 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, inject, OnInit } from '@angular/core';
 import { RouterOutlet } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { SectionActions } from '../section';
 
 @Component({
   selector: 'app-home',
   standalone: true,
   imports: [RouterOutlet],
   template: `
     application shell
+
     <router-outlet></router-outlet>
   `,
-  styles: [],
+  styles: [``],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class HomeComponent {}
+export class HomeComponent implements OnInit {
+  private sectionStore = inject(Store);
+
+  ngOnInit() {
+    this.loadSections();
+  }
+
+  public loadSections() {
+    this.sectionStore.dispatch(SectionActions.getSections());
+  }
```

Comment:
ten kod jest potrzebny? A jeśli jest to jeśli i tak loadSections jest publiczne to wrzuć to `this.sectionStore.dispatch(SectionActions.getSections())` od razu do ngOnInit